po/sv.po

```diff
@@ -568,290 +573,291 @@ msgstr "Kategori"
 #. None,
 #. ()),
 #. }
-#: ../src/gourmet/ui/shopCatEditor.ui.h:2 ../src/gourmet/reccard.py:2170
-#: ../src/gourmet/reccard.py:2230
-#: ../src/gourmet/plugins/key_editor/keyEditor.py:510
-#: ../src/gourmet/plugins/key_editor/recipeEditorPlugin.py:115
-#: ../src/gourmet/plugins/nutritional_information/nutritionInfoEditor.py:192
+#: ../src/gourmand/ui/shopCatEditor.ui.h:2 ../src/gourmand/reccard.py:2138
+#: ../src/gourmand/reccard.py:2198
+#: ../src/gourmand/plugins/key_editor/keyEditor.py:510
+#: ../src/gourmand/plugins/key_editor/recipeEditorPlugin.py:116
+#: ../src/gourmand/plugins/nutritional_information/nutritionInfoEditor.py:192
 msgid "Key"
 msgstr "Tangent"
 
-#: ../src/gourmet/ui/shopCatEditor.ui.h:3
+#: ../src/gourmand/ui/shopCatEditor.ui.h:3
 msgid "Category Editor"
 msgstr "Kategoriredigerare"
 
-#: ../src/gourmet/ui/shopCatEditor.ui.h:4
+#: ../src/gourmand/ui/shopCatEditor.ui.h:4
 msgid "Search by"
-msgstr ""
+msgstr "Sök _i"
```

Comment:
What's the motivation for this `_`?

Reply:
None (probably the editor that suggested it from the string "Search _in" and then I was unfocused)